### ISSUE TITLE
Design Structure

### ISSUE THREAD

**Issue by saqibdb** [Saqib-db/Reminder_iOS#3]
I have made a design structure, Check it if it is ok
- need some flat icons to go with the design, check to see which can be used
  ![screen shot 2015-06-10 at 10 32 51 am](https://cloud.githubusercontent.com/assets/5718485/8075001/1906f9d4-0eef-11e5-8840-e8aea19d0c64.png)
  ![screen shot 2015-06-10 at 10 32 41 am](https://cloud.githubusercontent.com/assets/5718485/8075003/1a801138-0eef-11e5-9b36-82354b1def42.png)

**Comment by moranciejr** [comment]
Not bad.. it's a start....but I dont think "time wasters" such as twitter and facebook should be in this app though. Lol..we could add contacts, Calendar, Important Events

**Comment by saqibdb** [comment]
@moranciejr Yes, twitter and facebook was just there as a dummy buttons.... Anyways I have also implemented the calendar.....
here is the screenshot
![screen shot 2015-06-13 at 12 23 36 am](https://cloud.githubusercontent.com/assets/5718485/8137065/8862bc34-10f5-11e5-80ec-0b46068ebfb8.png)

Its initial design and color scheme......
Also do let me know whenever you want to test the application on cell..... Its a hectic and a bit time consuming process .

**Comment by moranciejr** [comment]
Here area a few icons.  What do you need me to find?  Exactly what icons?
Do you know where to find them?  I think flat icons will be a nice addition
for thea app.  Your thought?

**Comment by saqibdb** [comment]
@moranciejr No need to worry about icons, that was meant for Muhd, as he is designing some icons for the application........ If you have any preference on any type of icons then do let us know, we will make something like you want

**Comment by Umer-naseer** [comment]
@saqibdb @moranciejr 
just add menu icon in app 
![main_page_icon](https://cloud.githubusercontent.com/assets/10264004/8144296/4be2ffa2-11f0-11e5-8f2b-4c1c25d5e998.png)
![main_menu](https://cloud.githubusercontent.com/assets/10264004/8144298/5a590ff4-11f0-11e5-8def-9c9c51c4b819.png)

**Comment by moranciejr** [comment]
It quite difficult to understand the flow.. Maybe its all the colors...Maybe we dont need all the colors in the app.  Please confer with the designer on the colors.  I like the colors but its making the app look busy and hard for me to follow. How do you think everything is coming?
